src/telemetry/metrics/metrics-setup.ts

```diff
@@ -0,0 +1,45 @@
+import { OTLPMetricExporter } from '@opentelemetry/exporter-metrics-otlp-http';
+import { MeterProvider, PeriodicExportingMetricReader } from '@opentelemetry/sdk-metrics';
+import { ATTR_SERVICE_NAME, ATTR_SERVICE_VERSION } from '@opentelemetry/semantic-conventions';
+import { resourceFromAttributes } from '@opentelemetry/resources';
+import { Counter } from '@opentelemetry/api';
+import { TelemetryConfig } from '../acrolinxInstrumentation';
+
+export const setupMetrics = (config: TelemetryConfig) => {
+  const collectorOptions = {
+    url: `${config.acrolinxUrl}/otlp/metrics`,
+    headers: {
+      Authorization: `Bearer ${config.accessToken}`,
+    },
+  };
+  const metricExporter = new OTLPMetricExporter(collectorOptions);
+
+  const resource = resourceFromAttributes({
+    [ATTR_SERVICE_NAME]: config.serviceName,
+    [ATTR_SERVICE_VERSION]: config.serviceVersion,
```

Comment:
What is serviceName and serviceVersion here? How its being used?

Reply:
It's the appName and app version that we ant to monitor. The open telemetry terminology calls it service.
It is part of the semantic conventions. Comes from here: https://github.com/open-telemetry/opentelemetry-js/blob/main/semantic-conventions/src/stable_attributes.ts#L645